data_write/data_write.py

```diff
@@ -887,20 +1124,21 @@ def write_tx_data():
 
         if write_rawacf:
             #procs.append(threading.Thread(target=do_acf))
+            write_correlations(copy.deepcopy(parameters_holder))
             pass
 
         if write_bfiq and data_parsing.bfiq_available:
             #procs.append(threading.Thread(target=write_bfiq_params, args=(parameters_holder.copy(), )))
-            write_bfiq_params(parameters_holder.copy())
+            write_bfiq_params(copy.deepcopy(parameters_holder))
```

Comment:
Is this dep copy required due to the possibility of writing multiple file types at once?

Reply:
is required, without deepcopy all writing algorithms point to same object